docs-source/content/userguide/managing-domains/domain-lifecycle/introspection.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Rerunning introspection"
+date: 2020-06-04T08:14:51-05:00
+draft: true
+weight: 5
+description: "This document describes rerunning the introspector job in the Oracle WebLogic Server in Kubernetes environment."
+---
+
+
+This document describes _how and when_ to rerun the introspector job in the Oracle WebLogic Server in Kubernetes environment.
+
+#### Overview
+
+In order to manage a domain, the operator needs to know its definition in WebLogic Server. It does this by running an _introspector job_
```

Comment:
Trying to imagine a customer reading this, I'm not sure what it means for the operator to "know its definition in WebLogic Server." Have we described this other places? What about "the operator needs to analyze the domain configuration."?

Reply:
Another suggestion is, "the operator needs to examine the domain configuration."